### PR TITLE
Issue 765 remove desc

### PR DESCRIPTION
**Pull request by ArthurData**
I noticed that {desc} is also used in `check_name_consistency()` > `document_and_reload()`. 
The issue only mentioned `fill_desc()` function. 

I keep this PR as a draft until your return ;)

### CODE REVIEW

**Comment by ColinFay** [comment]
@ArthurData For `check_name_consistency()`, we can switch to `as.data.frame(read.dcf("DESCRIPTION"))$Package`

File: R/reload.R [hunk 1]
```diff
@@ -36,8 +36,12 @@ check_name_consistency <- function(
 ){
   
   old_dir <- setwd(pkg)
-  
+
+  check_is_installed("desc")
+  required_version("desc", "1.4.0")
+
   package_name <- desc::desc_get("Package")
```
Comment (ColinFay) [inline R/reload.R:43]:
```r
desc_path <- file.path(pkg, "DESCRIPTION")
check_file_exist(desc_path)
package_name <- as.data.frame(read.dcf("DESCRIPTION"))$Package
```

Comment (ColinFay) [inline R/reload.R:43]:
```r
desc_path <- file.path(pkg, "DESCRIPTION")
check_file_exist(desc_path)
package_name <- as.data.frame(read.dcf("DESCRIPTION"))$Package
```

File: R/desc.R [hunk 2]
```diff
@@ -9,114 +9,123 @@
...
+  author_email,
+  author_orcid = NULL,
   repo_url = NULL,
   pkg = get_golem_wd()
-){
-  
+) {
+
+  check_is_installed("desc")
+  required_version("desc", "1.4.0")
```
Comment (ColinFay) [inline R/desc.R:32]: I don't think there is a need for a minimum version

**Review by ColinFay — COMMENTED** [review]
Add also skip_if_not_installed in test-desc.R

File: R/reload.R (hunk 1 above)
Comment (ArthurData) [reply R/reload.R:43]: Thank you @ColinFay

**Comment by ColinFay** [comment]
Note : I will have a look on Friday, 19th of November

**Comment by ColinFay** [comment]
We'll have this one merged for v 0.4.0

**Comment by ColinFay** [comment]
moved to another PR

**Pull request closed without merging**